ui/lib/pki/addon/components/page/pki-configure-create.hbs

```diff
@@ -90,14 +90,9 @@
     @message="To see configuration options, choose your desired output above."
     data-test-configuration-empty-state
   />
-  <div class="field is-grouped box is-fullwidth is-bottomless">
-    <div class="control">
-      <button type="button" class="button is-primary" disabled={{true}} data-test-pki-config-save>
-        Done
-      </button>
-      <button type="button" class="button has-left-margin-s" {{on "click" @onCancel}} data-test-pki-config-cancel>
-        Cancel
-      </button>
-    </div>
-  </div>
+  <hr class="has-background-gray-100" />
+  <Hds::ButtonSet>
+    <Hds::Button @text="Done" disabled={{true}} data-test-pki-config-save />
```

Comment:
I'm confused why this was originally always disabled?? Seems like the button should just be removed all together but not necessary to address in this PR

Reply:
It's because of the view. If you select a form type, then the form type field shows. It's basically a placeholder. I had the same thought. 
![image](https://github.com/hashicorp/vault/assets/6618863/5492ffa5-cd65-4598-9ff9-0452d509e113)